src/Button.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  display: flex;
+  background-color: #a0eaa3;
+  border-radius: 15px;
+  padding: 20px;
+  width: 100%;
+  max-width: 250px;
+  height: 30px;
+  color: #757775;
+  border-color: #d9e5da;
+  border-style: solid;
+  border-radius: 25px;
+  align-self: flex-end;
+  align-items: center;
+  justify-content: center;
```

Comment:
you probably don't need the flexbox here since buttons usually only have text as content, even a `Link` component from `react-router` basically just text with an anchor tag (`<a></a>`)